benchmark/compare.js

```diff
@@ -13,11 +14,12 @@ const cli = CLI(`usage: ./node compare.js [options] [--] <category> ...
   The output is formatted as csv, which can be processed using for
   example 'compare.R'.
 
-  --new    ./new-node-binary  new node binary (required)
-  --old    ./old-node-binary  old node binary (required)
-  --runs   30                 number of samples
-  --filter pattern            string to filter benchmark scripts
-  --set    variable=value     set benchmark variable (can be repeated)
+  --new      ./new-node-binary  new node binary (required)
+  --old      ./old-node-binary  old node binary (required)
+  --runs     30                 number of samples
+  --filter   pattern            string to filter benchmark scripts
+  --set      variable=value     set benchmark variable (can be repeated)
+  --progress true|false         show benchmark progress indicator
```

Comment:
What if we just make it so that it can be enabled with just `--progress`? We may need to specially check for it in the arg parsing since it doesn't have a value associated with it, I'm not sure.